apps/cookbook/src/app/page/side-nav/side-nav.component.html

```diff
@@ -49,6 +49,7 @@
       <li><a routerLink="showcase/slide-button" [routerLinkActive]="'is-selected'" class="side-nav__item">Slide Button</a></li>
       <li><a routerLink="showcase/calendar" [routerLinkActive]="'is-selected'" class="side-nav__item">Calendar</a></li>
       <li><a routerLink="showcase/checkbox" [routerLinkActive]="'is-selected'" class="side-nav__item">Checkbox</a></li>
+      <li><a routerLink="showcase/radio" [routerLinkActive]="'is-selected'" class="side-nav__item">Radio</a></li>
```

Comment:
Maybe we should consider having a separate nav-grouping for "form" controls, to make those easy to find?

Reply:
Agree - let's do that in a separate PR 👍